rules/Symfony64/Rector/Class_/FixImportWhileUsingRouteAttributeButAnnotationImportRector.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Symfony\Symfony64\Rector\Class_;
+
+use PhpParser\Node;
+use PhpParser\Node\Attribute;
+use Rector\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\Symfony\Tests\Symfony64\Rector\Class_\FixImportWhileUsingRouteAttributeButAnnotationImportRector
+ */
+final class FixImportWhileUsingRouteAttributeButAnnotationImportRector extends AbstractRector
+{
+    private const ANNOTATION_ROUTE = 'Symfony\Component\Routing\Annotation\Route';
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition(
+            'Replace Symfony\Component\Routing\Annotation\Route by Symfony\Component\Routing\Attribute\Route when the class use #Route[] attribute',
+            [
+                new CodeSample(
+                    <<<'CODE_SAMPLE'
+    use Symfony\Component\Routing\Annotation\Route;
+
+    /**
+     * #[Route("/foo")]
+    */
+    public function create(Request $request): Response
+    {
+        return new Response();
+    }
+CODE_SAMPLE
+                    ,
+                    <<<'CODE_SAMPLE'
+    use Symfony\Component\Routing\Attribute\Route;
+
+    #[Route('/foo')]
+    public function create(Request $request): Response
+    {
+        return new Response();
+    }
+CODE_SAMPLE
+                ),
+
+            ]
+        );
+    }
+
+    public function getNodeTypes(): array
+    {
+        return [Attribute::class];
```

Comment:
use `ClassMethod` instead, and rename its `attrGroups` so it only change under `ClassMethod`

Reply:
The #Route[] attribute can be specified on a method but also on a class in Symfony

Reply:
then, you need to define in `Class_` and `ClassMethod`